FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.E2ETests/E2ETests.cs

```diff
@@ -70,7 +72,9 @@ public async Task GivenRecentDateRange_WhenProcess_CorrectResultShouldBeReturned
 
             // Load configuration and set endTime to yesterday
             var configuration = new ConfigurationBuilder().AddJsonFile(_configurationPath).Build();
-            configuration.GetSection(ConfigurationConstants.JobConfigurationKey)["endTime"] = DateTime.Now.Date.ToString("yyyy-MM-dd");
+            var now = DateTime.Now;
+            configuration.GetSection(ConfigurationConstants.JobConfigurationKey)["startTime"] = now.AddMinutes(-1 * _triggerIntervalInMinutes).ToString(_dateTimeFormat);
+            configuration.GetSection(ConfigurationConstants.JobConfigurationKey)["endTime"] = now.ToString(_dateTimeFormat);
```

Comment:
Should sub minutes for "endTime"?

Reply:
End time will be subtracted in our codes before sending the request to FHIR Server, so we don't need to subtract it here.